src/js/config.js

```diff
@@ -20,50 +18,15 @@ export const MESSAGES = {
 	MSG_4250: '<p>You do not have rights to republish this type of content. Please contact your Account Manager for further details.</p>',
 	MSG_4300: '<p>Report contains multiple articles. Please view each article individually for republishing rights.</p>',
 	MSG_5000: '<p>Sorry, an error has occurred. Please try signing out and then in again. If error persists, please contact your Account Manager.</p>',
-	MSG_5100: '<p>The Republishing Service is currently undergoing maintenance. Please try again later.</p>'
+	MSG_5100: '<p>The Republishing Service is currently undergoing maintenance. Please try again later.</p>',
+	GRAPHICS: 'Please note that for copyright reasons not all the graphics in this article are available for republishing so will not be included in the download.',
+	WORD_FORMAT: 'Graphics are only available in Word format. Please select Word format (docx) in your  <a data-trackable="contractual-rights" href="/republishing/contract">syndication management tool</a> to download.'
```

Comment:
All the other messages are wrapped in `<p>`, so should GRAPHICS and WORD_FORMAT be wrapped in `<p>` too?

Reply:
No, the mark up for where those messages are used is taken care of by o-message. In fact this is exactly the type of thing that prompted me to suggest we refactor n-syndication to use jsx templating. It's easier to keep mark up and content seprate.